README.md

```diff
@@ -53,10 +53,20 @@ Take a look at `config.py`.  There are two settings which are pretty self-explan
 # Set to True to use random pages instead of curated selections in PAGES
 RANDOM = True
 
+# Set to True to enable the 'This day in history' function, given that RANDOM is False
```

Comment:
These additions make the `There are two settings...` text above confusing. I think things should be explained that there are three ways to use the plugin:
1. Random from all pages
2. Random from the current day of the year across a chosen range of years
3. Hand-selected

My initial thought is the general explanations with abbreviated names for each behavior should be in the first paragraph. This section here should list the abbreviated names with the settings required / relevant to each, perhaps with examples and re-iteration of the resulting behavior.

Reply:
Excellent re-write! Much more clear what config is required for each behavior. Thanks!